event/event_test.go

```diff
@@ -110,58 +65,57 @@ func (hub *testLabHub) GetLabByTag(string) (lab.Lab, error) {
 	return nil, nil
 }
 
-type testDockerHost struct {
-	docker.Host
-}
-
-func (dh *testDockerHost) GetDockerHostIP() (string, error) {
-	return "1.2.3.4", nil
-}
-
-type testEventFile struct {
-	store.EventFile
-}
+func TestEvent_StartAndClose(t *testing.T) {
+	dialer, close := store.CreateTestServer()
+	defer close()
+
+	conn, err := grpc.DialContext(context.Background(), "bufnet",
+		grpc.WithDialer(dialer),
+		grpc.WithInsecure(),
+	)
+	if err != nil {
+		t.Fatalf("failed to dial bufnet: %v", err)
+	}
+	defer conn.Close()
 
-func (ef *testEventFile) GetTeams() []store.Team {
-	return []store.Team{}
-}
+	client := pb.NewStoreClient(conn)
 
-func TestEvent_StartAndClose(t *testing.T) {
 	tt := []struct {
 		name string
 	}{
 		{name: "Normal"},
 	}
-
 	for _, tc := range tt {
 		t.Run(tc.name, func(t *testing.T) {
-			ctfd := testCtfd{}
 			guac := testGuac{}
 			hub := testLabHub{}
-			store := testEventFile{}
+
+			ts, _ := store.NewEventStore(store.EventConfig{
+				Name:           "Test Event",
+				Tag:            "test",
+				Available:      1,
+				Capacity:       2,
+				Lab:            store.Lab{},
+				StartedAt:      nil,
+				FinishExpected: nil,
+				FinishedAt:     nil,
+			}, client)
 
 			ev := event{
-				ctfd:    &ctfd,
 				guac:    &guac,
 				labhub:  &hub,
-				closers: []io.Closer{&ctfd, &guac, &hub},
-				store:   &store,
+				closers: []io.Closer{&guac, &hub},
+				store:  ts,
 			}
 
 			ev.Start(context.Background())
 
-			if ctfd.status != STARTED {
-				t.Fatalf("Expected CTFd to be started, but hasn't")
-			}
 			if guac.status != STARTED {
 				t.Fatalf("Expected Guacamole to be started, but hasn't")
 			}
 
 			ev.Close()
 
-			if ctfd.status != CLOSED {
-				t.Fatalf("Expected CTFd to be stopped, but hasn't")
-			}
 			if guac.status != CLOSED {
 				t.Fatalf("Expected Guacamole to be stopped, but hasn't")
 			}
```

Comment:
I have to check it out. The previous test was similar to this one.

Reply:
Let me know whether you will gonna change it or not. (I think, it is good to use `tt` tables if there are more than one test cases, otherwise it can be tested without `tt` tables, or it is much better to include different scenarios for test purposes.  )